csharp/ql/src/change-notes/2022-02-23-extractor-option-compression.md

```diff
@@ -0,0 +1,5 @@
+---
+category: minorAnalysis
+---
+* The C# extractor now accepts an extractor option `trap.compression`, which is used to decide the compression format for TRAP files. The legal options are `brotli` (default), `gzip` or `none`.
```

Comment:
Perhaps add: ``The option is added via `codeql database create --language=csharp -Otrap.compression=value ...` ``